R/create_map_measurements.R

```diff
@@ -0,0 +1,24 @@
+#' Create map
```

Comment:
A good start, yet I feel this function does more, such as 'Create a map with car traffic densities'. I suggest you update this to a more descriptive text. Note that I will ask the same for the function's name below :+1: